modules/pel/peltool/peltool.py

```diff
@@ -588,7 +607,7 @@ def main():
     parser.add_argument('-D', '--delete-all', dest='deleteAll',
                         action='store_true', help='Delete all PELs')
     parser.add_argument('-S', '--severities', nargs='+', choices=list(severityGroupValues.keys()), 
-                        help=f'Filter by severity value, consider pels matching the provided severities. {list(severityGroupValues.keys())}')
+                        help=f'Filter by severity value, consider pels matching the provided severities.')
```

Comment:
I think you could probably leave out the 'consider pels matching the provided severities' since I think the 'Filter by severity value' covers that.